README.md

```diff
@@ -10,10 +10,12 @@ Here is the upcoming release schedule for Service Fabric runtime versions that w
 
```

Comment:
Should update the top sentence to indicate that these are dates starting with 8.0. I may also specify that these are estimates and subject to change.

Reply:
done.